timepiece/crm/forms.py

```diff
@@ -135,6 +135,30 @@ def clean(self):
                     self._errors['client_primary_poc'] = self.error_class(
                         ['You must select a Contact that belongs to the selected Company.'])
 
+            target_internal_completion_date = cleaned_data.get('target_internal_completion_date')
+            required_completion_date = cleaned_data.get('required_completion_date')
+            target_open_date = cleaned_data.get('target_open_date')
+            start_date = cleaned_data.get('start_date')
+            turn_in_date = cleaned_data.get('turn_in_date')
+
+            if start_date:
+                if target_internal_completion_date and target_internal_completion_date < start_date:
+                    self._errors['target_internal_completion_date'] = self.error_class(
```

Comment:
It is not good practice to directly interact with private variables (that's what the _ indicates).  I suggest you update this code to use this function: https://docs.djangoproject.com/en/1.8/ref/forms/api/#django.forms.Form.add_error